src/index.html

```diff
@@ -4,65 +4,375 @@
 <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Stake'n'Vote</title>
+    <title>Kusamarian Stake'n'Vote</title>
     <style>
         @import "/src/styles.css";
     </style>
 </head>
-<body>
+<body onload="preLoad()">
     <nav class="navbar">
         <div class="navbar-brand">
-            <a class="navbar-item is-size-3 has-text-weight-semibold" href="#">Stake'n'Vote</a>
+            <a class="navbar-item is-size-3 has-text-weight-semibold" href="#"><img class="logo" src="logos/kulogo.png" alt=""></a>
         </div>
         <div class="navbar-end">
             <my-address-chooser id="address-chooser"></my-address-chooser>
         </div>
     </nav>
+
+    <!-- -------------- justadev ------------------------- -->
```

Comment:
Well, I'd rather not have that directly in the file — we have `git blame` and I think that's enough.
If you'd like to commemorate our contributions somewhere, may I suggest just adding [humans.txt](https://humanstxt.org/) to the project?